node/src/chain/acceptor.rs

```diff
@@ -101,6 +101,12 @@ impl ProvisionerChange {
     }
 }
 
+pub static DUSK_KEY: LazyLock<PublicKey> = LazyLock::new(|| {
```

Comment:
Isn't this duplicated from `rusk-recovery`? Couldn't it be gotten from there?

Reply:
`node` has no `rusk-recovery` dependency

Reply:
Seems like an opportunity to introduce it